main.go

```diff
@@ -170,6 +170,11 @@ func main() {
 			log.Printf("Listing workflow runs for: %s", repo.GetFullName())
 			if orgName != "" {
 				runs, res, err = client.Actions.ListRepositoryWorkflowRuns(ctx, orgName, repo.GetName(), opts)
+				if err != nil {
```

Comment:
I don't think this is a good fix.

You need to look at the specific error and HTTP code and determine if the problem is due to actions being disabled on the repo or something else, and if it's something else, it must continue to fail.

Reply:
I've tested this on repos which have actions disabled and get no errors.

Can you share the exact console output and how to reproduce it?